02_maps/08/main.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+func main() {
+
+	arr1 := []int{1, 2, 3, 4, 5, 4, 222}
+	arr2 := []int{4, 1, 2, 3, 5, 5, 222}
+
+	println(isEqualArrays(arr1, arr2))
+}
+
+func isEqualArrays[T comparable](arr1 []T, arr2 []T) bool {
+	resultingMap := make(map[T]bool)
+
+	for i := range arr1 {
+		resultingMap[arr1[i]] = false
+	}
+
+	for i := range arr2 {
+		_, ok := resultingMap[arr2[i]]
+		if !ok {
+			return false
+		} else {
+			resultingMap[arr2[i]] = true
+		}
```

Comment:
можно и не делать лишнюю лесенку с else
```suggestion
		if !ok {
			return false
		}
		resultingMap[arr2[i]] = true
```